packages/angular/src/ng-add-setup-project/rules/ng-module.ts

```diff
@@ -34,7 +32,6 @@ export function updateNgModule(
 
     const changes = [
       ...injectInitService(source, appModulePath),
```

Comment:
Have you tried all injecting the service in the `coveo.components.module.ts` file as well?